Content.Client/UserInterface/Systems/Scoreboard/ScoreboardUIController.cs

```diff
@@ -0,0 +1,64 @@
+using Content.Client.RoundEnd;
+using Robust.Client.Input;
+using Content.Shared.Input;
+using Robust.Shared.Input.Binding;
+using Content.Client.GameTicking.Managers;
+using Content.Client.Lobby;
+using Content.Client.Gameplay;
+using Robust.Shared.Log;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controllers;
+
+namespace Content.Client.UserInterface.Scoreboard
+{
+    public sealed class ScoreboardUIController : UIController, IOnStateEntered<LobbyState>, IOnStateEntered<GameplayState>, IOnStateExited<LobbyState>, IOnStateExited<GameplayState>
+    {
+
+        [Dependency] private readonly IInputManager _input = default!;
+        [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
+        private ClientGameTicker _gameTicker = default!;
+
+        public void OnStateEntered(LobbyState state)
+        {
+            HandleStateEntered();
+        }
+
+        public void OnStateEntered(GameplayState state)
+        {
+            HandleStateEntered();
+        }
+
+        public void OnStateExited(LobbyState state)
+        {
+            HandleStateExited();
+        }
+
+        public void OnStateExited(GameplayState state)
+        {
+            HandleStateExited();
+        }
+
+        private void HandleStateEntered()
+        {
+            _gameTicker = _entitySystem.GetEntitySystem<ClientGameTicker>();
+
+            _input.SetInputCommand(ContentKeyFunctions.OpenScoreboardWindow,
+                InputCmdHandler.FromDelegate(_ =>
+                {
+                    if (_gameTicker.Window != null)
+                        OpenScoreboardWindow(_gameTicker.Window);
+                }));
+        }
```

Comment:
Use UISystemDependency.

Reply:
Alright, I've spent enough hours slamming my head into walls today about this.

[UISystemDependency] is for systems, and [Dependency] is for controllers, according to your own documentation. 

https://docs.spacestation14.com/en/robust-toolbox/user-interface.html

Reply:
There are hundreds of examples in the code of [Dependency] being used with IEntitySystemManager and IInputManager, 

and not one example of [UISystemDependency] being used to access IEntitySystemManager or IInputManager

Please elaborate on what you want me to change about this working system because it is not clear.

Reply:
1. Use UISystemDependency where?
2. In what way? There's no examples of it being used for something like this in the entire codebase. 
3. Does this directly contradict with your later comment to just move the window over to the controller?

Reply:
Alright, you know what. I had it working fine, and after a day of trying to interpret 2 incomplete sentence lines of feedback requesting basically a full recoding, I'm more confused than when I started.

Elaborate on what you want me to do, accept it as is, or let it rot. I can't handle this.

Reply:
All this to make a fucking window re-open. Good god.

Reply:
`[UISystemDependency] private readonly ClientGameTicker _ticker = default;` or something

Reply:
Everything all good mate?